### ISSUE TITLE
Cannot Train the model.

### ISSUE THREAD

**Issue by kunalgupta777** [JoshuaPiinRueyPan/ViolenceDetection#18]
Hi.
I have been trying to train the model using the Hockey dataset. But after the validation phase, my system just hangs and stays like that for 30 minutes and beyond. I had to force restart it.
Can you provide us with the .cpkt files  that can be used to run Deploy script?
Thanks.

My system specs are:
Intel Core i5 7th Gen
8GB DDR4 RAM
Nvidia GeForce 940MX ( although I am using CPU version of tensorflow )

**Comment by JoshuaPiinRueyPan** [comment]
Hi,
    I do not use the tensorflow-cpu so I can't imagine how long it would takes...
To me, I spend about 330 (s) to Train an epoch, and about 100 (s) to evaluate the validation set.  To evaluate the Test Set, it cost me 400 (s).  Could you use these info to extrapolate the testing time?  Also, is the CPU and Memory still high when you on such case?
